e2e/support/helpers/e2e-models-metadata-helpers.js

```diff
@@ -34,8 +36,13 @@ export function setColumnType(oldType, newType) {
     .contains(oldType)
     .click();
 
-  cy.get(".ReactVirtualized__Grid.MB-Select").scrollTo("top");
-  cy.findByPlaceholderText("Search for a special type").type(newType);
+  const scrollTimes = 3;
+  _.times(scrollTimes, () => {
```

Comment:
Nit (don't change it now):
Just for the future - Cypress already comes with Lodash so you can do `Cypress._.times()`

https://github.com/metabase/metabase/blob/master/docs/developers-guide/e2e-tests.md#cypress-comes-with-lodash-for-free

Reply:
@nemanjaglumac take another look - I removed `wait`